src/tribol/interface/tribol.cpp

```diff
@@ -391,6 +408,107 @@ void registerMesh( integer meshId,
 
 } // end of registerMesh()
 
+//------------------------------------------------------------------------------
+void registerMfemMesh( integer cs_id,
+                      integer mesh_id_1,
+                      integer mesh_id_2,
+                      mfem::ParMesh& mesh,
+                      const mfem::ParGridFunction& current_coords,
+                      const std::set<integer>& attributes_1,
+                      const std::set<integer>& attributes_2,
+                      integer contact_mode,
+                      integer contact_case,
+                      integer contact_method,
+                      integer contact_model,
+                      integer enforcement_method,
+                      integer binning_method)
+{
+   std::unique_ptr<mfem::FiniteElementCollection> dual_fec = nullptr;
+   integer dual_vdim = 0;
+   auto mfem_data = std::make_unique<MfemMeshData>(
+      mesh_id_1,
+      mesh_id_2,
+      mesh,
+      current_coords,
+      attributes_1,
+      attributes_2
+   );
+   // register empty meshes so the coupling scheme is valid
+   registerMesh(
+      mesh_id_1, 0, 0, nullptr, mfem_data->GetElemType(), nullptr, nullptr);
```

Comment:
Does mfem_data->GetElemType() query the mfem mesh and use an mfem function to return an element type? I recall that the mfem element types and tribol element types do not align. Please verify.

Reply:
It returns the Tribol element type.